tests/Message/ItemDataTestTrait.php

```diff
@@ -13,7 +13,7 @@ protected function getExpectedOrderLine(): array
             'type' => 'shipping_fee',
             'name' => 'item-name',
             'quantity' => 1,
-            'tax_rate' => 200300,
+            'tax_rate' => 21,
```

Comment:
The tax rate was explicitly defined with 2 decimals, to test/validate the case fixed in this PR:
https://github.com/MyOnlineStore/omnipay-klarna-checkout/pull/27/files

Though a comment would have been helpful. Perhaps you could add one?